test/dist_autograd_test.py

```diff
@@ -536,20 +576,21 @@ def _verify_backwards_remote(self, tensors, context_id, local_grads, *args):
         self.assertEqual(ngrads, len(grads))
 
 
-    @dist_init
+    @dist_init(setup_model_parallel=True)
     def test_backward_simple(self):
         # Run the same code locally and with dist autograd and verify gradients
         # are same.
         local_grads = None
         t1 = torch.rand((3, 3), requires_grad=True)
         t2 = torch.rand((3, 3), requires_grad=True)
-        for exec_mode in [ExecMode.LOCAL, ExecMode.REMOTE]:
+        for exec_mode in [ExecMode.LOCAL, ExecMode.RPC_SYNC, ExecMode.REMOTE]:
             with dist_autograd.context() as context_id:
                 ret = self._exec_func(exec_mode, torch.add, t1, t2)
                 loss = ret.sum()
-                local_grads = self._verify_backwards(exec_mode, [loss], context_id, local_grads, t1, t2)
+                ret = self._verify_backwards(exec_mode, [loss], context_id, local_grads, t1, t2)
```

Comment:
Shouldn't we also verify here (and in other tests) that a gradient is accumulated on the OwnerRRef?

Reply:
The result tensor of `torch.add` (owner) actually does not require grad, but let me add another test to cover that.